user-defined-variables: Add note about SELECT...INTO <var> (#5268)
pingcap/docs: user-defined-variables.md

+------+
```

## MySQL compatibility

> The `SELECT ... INTO <variable>` syntax is supported in MySQL but not in TiDB. Other than this the syntax supported by MySQL and TiDB is identical. 

Reviewer: ```suggestion
Except for `SELECT ... INTO <variable>`, the syntax supported in MySQL and TiDB is identical.
```

+------+
```

> ## MySQL Compatibility

Reviewer 2: ```suggestion
## MySQL compatibility
```

The style guide for H2s and below is to only upper the first word.

The `SELECT ... INTO <variable>` syntax is supported in MySQL but not in TiDB.

> Other than this the syntax supported by MySQL and TiDB is identical. 

Reviewer 2: The linter is complaining about this because it is a manual line break. Please either remove the break or use bullet points.
